docs/proposals/backup-restore.md

```diff
@@ -260,6 +263,42 @@ registry-b.k8s.io/kube-apiserver@sha256:9081a6f83f4febf47369fc46b6f0f7683c7db243
             └── io.cncf.notary.x509chain.thumbprint#S256: '["xxxxxx"]'
 ```
 
+Back up an image from Docker Hub to local. Upon success, the output in a non-tty mode will be:
```

Comment:
The `non-tty` mode is for plaintext, which means it only allows ASCII characters. Characters likes `•`, `├─`, and `✔` (emojis) are not allowed.

Reply:
I removed the verbose output changes. ORAS maintainers aligned that we will keep the verbose output as it is now since  verbose output it's not for terminal users.